Sources/AltSwiftUI/Source/Views/ReadOnly/Label.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  Label.swift
+//  AltSwiftUI
+//
+//  Created by Chan, Chengwei on 2021/05/24.
+//
+
+import UIKit
+
+@available(iOS 14.0, *)
+public typealias LocalizedStringKey = String
+
+/// A view that display icon with text.
+public struct Label<Title: View, Icon: View>: View {
+    public var viewStore = ViewValues()
+    var title: Title
+    var icon: Icon
+    
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The visual representation of right part text of the label
+    ///     - icon: The visual representation of left part image of the label
+    public init(@ViewBuilder title: () -> Title, icon: () -> Icon) {
+        self.title = title()
+        self.icon = icon()
+    }
+
+    public var body: View {
+        self
+    }
+}
+
+extension Label where Title == Text, Icon == Image {
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The string of right part text of the label
+    ///     - icon: The visual representation of left part image of the label
+    public init<S: StringProtocol>(_ title: S, image: String) {
+        self.title = Text(title)
+        self.icon = Image(image)
+    }
+    
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The LocalizedStringKey of right part text of the label
+    ///     - icon: The visual representation of left part image of the label
+    @available(iOS 14.0, *)
```

Comment:
Why are these 3 inits iOS14 available? I think you should be able to remove this constraint.